me.mitya57.ReText.json

```diff
@@ -0,0 +1,382 @@
+                        "sed -r -i 's/^(([[:blank:]]*)(if ReTextWebKitPreview and globalSettings\\.useWebKit:)([[:blank:]]*))$/\\2return ReTextPreview\\(self\\)\\n\\1/' 'ReText/tab.py';",
+                        "echo 'ReText/window.py';",
+                        "diff 'ReText/window.py.bak' 'ReText/window.py' -C 4 || :;",
+                        "echo 'ReText/tab.py';",
+                        "diff 'ReText/tab.py.bak' 'ReText/tab.py' -C 4 || :;",
+                        "rm -f 'ReText/window.py.bak' 'ReText/tab.py.bak';"
+                    ]
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "echo 'all:\n\ninstall:\n\tpython3 setup.py build; python3 setup.py install -O1 --skip-build --prefix=/app --root=/;\n' > makefile;"
+                    ]
+                }
+            ],
+            "post-install": [
+                "echo 'post-install: manuals';",
+                "mkdir -p '/app/share/man/man1'; install -p -m 0644 'retext.1' '/app/share/man/man1/';",
+                "appname='retext';
+                mkdir -p 'man/de';
+                perllib_path=\"$( find po4a-*/lib -mindepth 0 -maxdepth 0 -xtype d | sort -V | head -n 1 )\";
+                echo \"${perllib_path}\";
+                PERLLIB=\"${perllib_path}\" ./po4a-0.52/po4a-updatepo -v -M utf-8 -f man -m 'retext.1' -p 'retext-man-de.po';
+                PERLLIB=\"${perllib_path}\" ./po4a-0.52/po4a-translate -M utf-8 -f man --option groff_code=verbatim -m 'retext.1' -p 'retext-man-de.po' -l \"man/de/${appname}.1\";
+                mkdir -p '/app/share/man/de/man1';
+                install -p -m 0644 \"man/de/${appname}.1\" '/app/share/man/de/man1/';",
+                "echo 'post-install: icons';",
+                "appname='retext';
+                imagemagick_home='/app/opt/imagemagick';
+                imagemagick_lib=\"${imagemagick_home}/lib\";
+                imagemagick_bin=\"${imagemagick_home}/bin\";
+                if [[ -n \"${LD_LIBRARY_PATH}\" ]]; then
+                  LD_LIBRARY_PATH=\"${LD_LIBRARY_PATH}:${imagemagick_lib}\";
+                else
+                  LD_LIBRARY_PATH=\"${imagemagick_lib}\";
+                fi;
+                export LD_LIBRARY_PATH;
+                if [[ -n \"${PATH}\" ]]; then
+                  PATH=\"${PATH}:${imagemagick_bin}\";
+                else
+                  PATH=\"${imagemagick_bin}\";
+                fi;
+                export PATH;
+                icon='icons/retext.svg';
+                for s in {16,22,24,32,48,64,72,96,128,192,256,512}; do
+                    size=\"${s}x${s}\";
+                    echo \"- ${size}\";
+                    mkdir -p \"icons/${size}/\";
+                    convert -background none -density 1024 -resize \"${size}\" \"${icon}\" \"icons/${size}/${appname}.png\";
```

Comment:
As mentioned in the other one `rsvg-convert` is in the sdk so you can drop the build dep.